web

```diff
@@ -1 +0,0 @@
-../traccar-web/web
```

Comment:
I'm running Windows.  When I clone the repository, this is not a link.  It's a regular file.  I'm not sure how to run the web interface with this file untouched.